Argo/Globals/fold.swift

```diff
@@ -0,0 +1,7 @@
+import Runes
+
+func foldM<S: SequenceType, U>(sequence: S, initial: U, combine: (U, S.Generator.Element) -> U?) -> U? {
```

Comment:
This is completely internal, so I'm not super duper worried about that, honestly.